mdf_forge/forge.py

```diff
@@ -437,6 +437,60 @@ def match_titles(self, titles):
         return self
 
 
+    def match_years(self, years=[], min = None, max = None, include=True):
```

Comment:
We should have the default for the `years` argument be `None` because mutable default arguments can cause unexpected behavior.

Also, I'm not sure I understand the difference between the `include` argument here and `match_range`'s `inclusive`. If there's no difference, we should use the same terms.

Reply:
Done